config/winston.js

```diff
@@ -0,0 +1,47 @@
+import appRoot from 'app-root-path';
+import winston from 'winston';
+
```

Comment:
I believe we can enhance the display by creating a custom function that adds the time and formats the output accordingly
```suggestion

const formatParams = (info) => {
  const {
    timestamp, level, message, ...args
  } = info;
  const currentTime = timestamp.slice(0, 19).replace('T', ' ');

  return `${currentTime} ${level}: ${message} ${
    Object.keys(args).length ? JSON.stringify(args, '', '') : ''
  }`;
};
```